### DISCUSSION TITLE
How do you add a theme to fish shell

### DISCUSSION THREAD

**Discussion by BrunoCBart** [fish-shell/fish-shell#8683] [Q&A] [answered]
I downloaded simnalamburt/shellder theme using fisher but I don't know how to add it to the shell can anyone help me?

**Comment by faho** [comment, marked as answer]
Fish has no specific support for "themes" (really we'd call them "prompts" probably). It's all just functions.

So: You take the function files like fish_prompt.fish and fish_right_prompt.fish and such, and drop them in ~/.config/fish/functions. That allows fish's autoloading mechanism to find these functions by name and use them as the prompt.

If you had a package manager that would be doing this for you.

----

I would argue against this particular theme because it's unmaintained, and I think I recall it being broken.
